modules/ray_tracing/doc/content/source/ray_kernels/LineSourceRayKernel.md

```diff
@@ -0,0 +1,30 @@
+# LineSourceRayKernel
+
+!syntax description /RayKernels/LineSourceRayKernel
+
+The strong form, given the line
+
+!equation
+L = \{\vec{r}_1 + t\vec{r}_2 \mid t \in [0, 1]\}\,
+
+is defined as
+
+!equation
+\underbrace{-f(r')}_{\text{LineSourceRayKernel}} + \sum_{i=1}^n \beta_i(u,r) = 0\,,\quad r\in\Omega\,,\quad r'\in L
+
+where $f$ is the source term (negative if a sink) along $L$ and the second term on the left hand side represents the strong forms of other kernels and boundary conditions. The `LineSourceRayKernel` weak form, in inner-product notation, is defined as
+
+!equation
+R_i(u_h) = (\psi_i, -f)_L \quad \forall ~ \psi_i,
+
+where the $\psi_i$ are the test functions, and $u_h$ are the trial solutions in the finite dimensional space $\mathcal{S}^h$ for the unknown ($u$).
+
+The Jacobian term for this kernel is zero: $\frac{\partial R_i(u_h)}{\partial u_j} = 0$, since it is assumed that $f$ **is not** a function of the unknown $u$.
```

Comment:
```suggestion
The Jacobian term for this kernel is zero: $\frac{\partial R_i(u_h)}{\partial u_j} = 0$, since it is assumed that $f$ +is not+ a function of the unknown $u$.
```

https://mooseframework.inl.gov/python/MooseDocs/extensions/core.html#text-formatting